libs/perun/pipes/src/lib/group-sync-icon.pipe.ts

```diff
@@ -24,14 +24,14 @@ export class GroupSyncIconPipe implements PipeTransform {
     if (normalSyncEnabled === null && structureSyncEnabled === null) {
       return '';
     }
-    const structureSyncEnabledValue = structureSyncEnabled.value as unknown as string;
-    const normalSyncEnabledValue = normalSyncEnabled.value as unknown as string;
+    const structureSyncEnabledValue = structureSyncEnabled.value as string;
```

Comment:
this was boolean before eslint
`const structureSyncEnabledValue = <boolean>(<unknown>structureSyncEnabled.value);`